keyboards/meishi2/keymaps/moc/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019 Biacco42
```

Comment:
This should refer to your actual name, not just to a user id.  Also, you might want to update the year.